code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -351,6 +351,26 @@
 	nutriment_factor = 5 * REAGENTS_METABOLISM
 	color = "#302000" // rgb: 48, 32, 0
 	taste_description = "bitterness"
+	
+/datum/reagent/consumable/coco/on_mob_life(mob/living/carbon/M)
+	if(iscatperson(M))
+		if(prob(40))
+			M.adjust_disgust(20)
+		if(prob(5))
+			M.visible_message("<span class='warning'>[M] [pick("dry heaves!","coughs!","splutters!")]</span>")
+		if(prob(10))
+			var/sick_message = pick("You feel nauseous.", "You're nya't feeling so good.","You feel like your insides are melting.","You feel illsies.")
```

Comment:
brilliant

Reply:
Im dyi-wying :3

Reply:
You feel like you could use some helpies.